### DISCUSSION TITLE
[Question][Hw4] Github Problem

### DISCUSSION THREAD

**Discussion by anisrashidov** [prosyslab-classroom/cs348-information-security#708] [Q&A]
I keep trying to push my modifications, and it says that it worked in my VS Code terminal, however, the modifications are not reflected in github. Instead, the code remained the same and the following icon appeared (screenshot). Also, if you notice, it says that only one commit has been done so far, but that's not true. I ask if anyone knows what the problem might be?!
![image](https://github.com/prosyslab-classroom/cs348-information-security/assets/89933766/818f6ee5-b4ff-42ef-826d-7b772ab6b110)

**Comment by anisrashidov** [comment]
Nevermind, problem solved.

**Comment by 727yubin** [comment]
Github is currently having difficulties. Try checking https://www.githubstatus.com/ for updates.
